app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+        padding: 8px 16px;  
+        &:hover{
+            background: $magenta_dark;
+            border: 3px solid $magenta_dark;
+        }
+        &:focus{
+            background-color: $yellow_secondary;
+            border: 3px solid $support_focus;
+            color: $text_secondary;
+        }
+        &:active{
+            background: $magenta_darker;
+            color: $white;
+            border: 3px solid $yellow_secondary;
+            box-shadow: 0px 3px 0px $white, inset 0px 5px 2px rgba(0, 11, 59, 0.25);
+        }
+    }
+    &.button--small{
+        padding: 11px 16px 0 16px;
+        &:focus{
+            color: $text_secondary!important;
+        }
+    }
+
+    &.button--secondary {
+        background: $white;
+        color: $magenta_primary;
+        font-size: 16px;
+        border: 1px solid $magenta_primary;
+        border-radius: $border_radius;
+        box-shadow: 0px 3px 0px rgba(0, 11, 59, 0.25);
+        padding: 12px 16px;
+
+        &:hover{
+            //background: $light_grey_2;
+            border: 1px solid $magenta_dark;
+        }
+        &:focus{
+            background-color: $yellow_secondary;
+            border: 3px solid $support_focus;
+            color: $text_secondary;
+        }
+        &:active{
+            font-weight: 600;
+            background: $light_grey_2;
+            color: $text_secondary;
+            border: 3px solid $support_focus;
+            box-shadow: 0px 3px 0px $white, inset 0px 5px 2px rgba(0, 11, 59, 0.25);
+        }
+    }
+}
+
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Form elements ----------------------------------------------------------------
+label{
+    font-family: 'Roobert-Regular', sans-serif;
+    font-size: 16px;
+    color: $text_secondary;
+}
+input[type=text],
+input[type=number],
+input[type=email],
+input[type=password], 
+textarea{
+    width: 285px;
+    height: 40px;
+    font-family: 'Roobert-Regular', sans-serif;
+    box-sizing: border-box;
+    background-color: $white;
+    border: 1px solid $input_border;
+    border-radius: $border_radius;
+    padding: 9px 12px 9px 12px;
+    &:focus{
+        border: 2px solid $support_focus;
+        border-radius: 0;
+        background-color: $yellow_secondary;
+        outline: 0;
+    }
+}
+textarea{
+    height: auto;
+}
+
+ 
+
+input[type=radio],
+input[type=checkbox] {
+    &:focus,
+    &:hover{
+        outline-color: $support_focus;
+    }
+    + label{
+        cursor: pointer;
+    }
+}
+
+
+select{
+    width: 285px;
+    background-color: $white;
+    -webkit-appearance: auto;
+    -moz-appearance: auto;
+    padding: 7px 12px 7px 12px;
+    height: auto;
+    border-radius: $border_radius;
+    &:focus{
+        border: 2px solid $support_focus;
+        border-radius: 0;
+        background-color: $yellow_secondary;
+        outline: 0;
+    }
+}
+
+fieldset{
+    legend{
+        font-family: 'Roobert-Bold', sans-serif;
```

Comment:
Can we define the fallback font within the `@font-face` declaration to avoid the need for duplication?